ludwig/schema/combiners/comparator.py

```diff
@@ -12,6 +14,13 @@
 class ComparatorCombinerConfig(BaseCombinerConfig):
     """Parameters for comparator combiner."""
 
+    def __post_init__(self):
```

Comment:
Perhaps better placed in the `config_validation.checks` suite. Or we should decentralize that suite when you have schema-specific requirements (e.g. see `check_sequence_concat_combiner_requirements`).

Reply:
I originally had it there, but it was very ugly and hacky. I actually much prefer having it here, to better encapsulate the validation logic. In the future, we can look to remove this by combining `num_fc_layers` and `fc_layers` into a single `oneOf` option.